### PR TITLE
Lora load eval

### PR DESCRIPTION
**Pull request by wanderingweights**
In this PR we are moving over to lora head swapping rather than downloading entire finetune model (when this is possible). 

We have a backup full model reload if a miner submits one to be evaluated (ready for pre-training). 

This is also tested and confirmed with multi-gpu

### CODE REVIEW

File: validator/tasks/task_prep.py [hunk 1]
```diff
@@ -17,6 +17,9 @@
 from validator.utils.logging import logger
 from validator.utils.minio import async_minio_client
 
+from validator.utils.logging import create_extra_log
+from validator.utils.logging import logger
```
Comment (besimray) [inline validator/tasks/task_prep.py:21]: Duplicate imports

Comment (besimray) [inline validator/tasks/task_prep.py:21]:
```suggestion

```

File: validator/utils/logging.py [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import json
 from contextvars import ContextVar
 from logging import Formatter
-from logging import Logger
```
Comment (besimray) [inline validator/utils/logging.py:4]: we need this import

File: validator/endpoints/tasks.py [hunk 3]
```diff
@@ -24,6 +24,7 @@
 from validator.core.dependencies import get_config
 from validator.core.models import NetworkStats
 from validator.core.models import RawTask
+from validator.core.models import TrainingTaskStatus
```
Comment (besimray) [inline validator/endpoints/tasks.py:27]: This one seems to be missing in the models?

**Comment by wanderingweights** [comment]
In this PR we are moving over to lora head swapping rather than downloading entire finetune model (when this is possible).

We have a backup full model reload if a miner submits one to be evaluated (ready for pre-training).

This is also tested and confirmed with multi-gpu

**Pull request merged**